src/rest/methods/user.ts

```diff
@@ -77,6 +80,12 @@ export type PartialUserPermission = Partial<IUserPermission>
 
 export type UserPermissionResult = Promise<IUserPermission>
 
+export const remapUserResponse = (user: any) => {
```

Comment:
Let's rename this to `remapUserResult` because that matches up more with our type names (e.g. `UserResult`)

```suggestion
export const remapUserResult = (user: any) => {
```